test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -6554,6 +6554,41 @@ public void PackCommand_PackReadme_FolderNested_Succeeds(char readmeSeparator)
             TestPackReadmeSuccess(testDirBuilder, $"docs{s}nested{s}readme.md");
         }
 
+        [Fact]
+        public void PackCommand_PackReadme_MissingReadme_Succeeds()
+        {
+            var nuspec = NuspecBuilder.Create();
+            var testDirBuilder = TestDirectoryBuilder.Create();
+            var s = Path.DirectorySeparatorChar;
+
+            nuspec
+                .WithPackageId("test")
+                .WithFile("dummy.txt");
+
+            testDirBuilder
+                .WithFile("dummy.txt", 6)
+                .WithNuspec(nuspec);
+
+            using (testDirBuilder.Build())
+            {
+                var nupkgPath = Path.Combine(testDirBuilder.BaseDir, $"{testDirBuilder.NuspecBuilder.PackageIdEntry}.{testDirBuilder.NuspecBuilder.PackageVersionEntry}.nupkg");
+
+                // Act
+                var result = CommandRunner.Run(
```

Comment:
Considering there a "unit" test in `NuGet.Packagaing.Test`, does this test prevent any other regression? Tests like this that run a process, wait for the process the finish, and then assert the outcome as significantly slower than tests that run code in the test host process. Plus, duplicated tests make CI run slower without providing benefit.

Reply:
Thanks for the comments!
Yes, I feel hesitate when writing the functional tests (NuGet commandline, and dotnet.intergration.tests), for the same reason.
It might be good to let people know how to trade off.
I removed the two functional tests.